lib/tasks/coverage.rake

```diff
@@ -1,5 +1,5 @@
 namespace :test do
-  task :coverage do
+  task coverage: :environment do
```

Comment:
Does loading the environment influence the coverage at all? I was under the impression that simplecov needed to be loaded before the application to corrrectly calculate the coverage.

Reply:
Hmmm I did this because rubocop yelled at me about it. I take your point about coverage tho. I'll look into this.